### ISSUE TITLE
`big` not working as expected with iTerm2 build 2.1.1

### ISSUE THREAD

**Issue by shawnyeager** [apaszke/termtile#3]
In iTerm2 build 2.1.1 under OS X 10.10.5, the command has no effect after issuing a `dr` or `lr`, and actually makes the window smaller following a `ll` or `rr`.

**Comment by shawnyeager** [comment]
Looking more closely, I realized that `big` is increasing the window size following a `dr` and `lr` but that it is not repositioning the window. As a result, the window is larger, but extends off-screen to the bottom and right.

**Comment by apaszke** [comment]
Yeah I know it extends off-screen. I'll probably make it keep the windows visible at all times.
What did you mean that it makes the window smaller following `ll` or `rr`? It makes it wider but shorter for me. It's just a matter of configuration. By default `big` is just an alias for "resize to 1000x600", but you can customize it by passing your own arguments to it.

It isn't relative at the moment. Do you think it's worth creating something like `bigger`, which makes the window a bit bigger?
